doc/functions/dockertools.xml

```diff
@@ -557,7 +557,7 @@ buildImage {
 
   <para>
    Creating base files like <literal>/etc/passwd</literal> or
-   <literal>/etc/login.defs</literal> are necessary for shadow-utils to
+   <literal>/etc/login.defs</literal> is necessary for shadow-utils to
```

Comment:
This was correct before.

Reply:
Isn't the 'necessary' here applying to 'Creating' rather than 'base files' ?

Reply:
My mistake.